interview-questions.md

```diff
@@ -8,32 +8,38 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the virtual DOM in React?
 
-Your answer:
+Your answer: The virtual Document Object Model in React is what is displayed on the website/web browser.
 
-Researched answer:
+
+Researched answer: The virtual DOM in React acts like a blueprint. It show a visual representation of the HTML DOM that was created.
```

Comment:
I would include the significance of the virtual DOM.  It is very important, so make sure you know it before an interview. :)